Traceback (most recent call last):
  File "/home/spr/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/spr/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/nbclient/client.py", line 1087, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/spr/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/spr/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib64/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/spr/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/nbclient/client.py", line 540, in async_execute
    await self.async_execute_cell(
  File "/home/spr/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/nbclient/client.py", line 832, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/spr/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/nbclient/client.py", line 740, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
for filename in filelist:
    image = CCDData.read(filename) #, unit="adu")
    z_image = ccdproc.subtract_bias(image, master_bias)
    name_of_file = f'z{image.header["FILENAME"]}'
    z_image.header['FILENAME'] = name_of_file
    nowtime = datetime.datetime.now().isoformat('T', timespec='seconds')
    z_image.header['HISTORY'] = f'{nowtime} astropy ccdproc substract_bias'
    z_image.header['HISTORY'] = 'using NOT2008/N1/zero_N1.fits master BIAS' 
    print(f'writting {name_of_file} in {directory}')
    z_image.write(directory / name_of_file,overwrite='yes')
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;32m<ipython-input-14-f9f55815f050>[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0;32mfor[0m [0mfilename[0m [0;32min[0m [0mfilelist[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m      2[0m     [0mimage[0m [0;34m=[0m [0mCCDData[0m[0;34m.[0m[0mread[0m[0;34m([0m[0mfilename[0m[0;34m)[0m [0;31m#, unit="adu")[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 3[0;31m     [0mz_image[0m [0;34m=[0m [0mccdproc[0m[0;34m.[0m[0msubtract_bias[0m[0;34m([0m[0mimage[0m[0;34m,[0m [0mmaster_bias[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      4[0m     [0mname_of_file[0m [0;34m=[0m [0;34mf'z{image.header["FILENAME"]}'[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m     [0mz_image[0m[0;34m.[0m[0mheader[0m[0;34m[[0m[0;34m'FILENAME'[0m[0;34m][0m [0;34m=[0m [0mname_of_file[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/ccdproc/log_meta.py[0m in [0;36mwrapper[0;34m(*args, **kwd)[0m
[1;32m     90[0m         [0;31m# Grab the logging keyword, if it is present.[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[1;32m     91[0m         [0mlog_result[0m [0;34m=[0m [0mkwd[0m[0;34m.[0m[0mpop[0m[0;34m([0m[0m_LOG_ARGUMENT[0m[0;34m,[0m [0;32mTrue[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 92[0;31m         [0mresult[0m [0;34m=[0m [0mfunc[0m[0;34m([0m[0;34m*[0m[0margs[0m[0;34m,[0m [0;34m**[0m[0mkwd[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     93[0m [0;34m[0m[0m
[1;32m     94[0m         [0;32mif[0m [0;32mnot[0m [0mlog_result[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/ccdproc/core.py[0m in [0;36msubtract_bias[0;34m(ccd, master)[0m
[1;32m    586[0m                                "image".format(ccd.unit, master.unit))
[1;32m    587[0m         [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 588[0;31m             [0;32mraise[0m [0me[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    589[0m [0;34m[0m[0m
[1;32m    590[0m     [0mresult[0m[0;34m.[0m[0mmeta[0m [0;34m=[0m [0mccd[0m[0;34m.[0m[0mmeta[0m[0;34m.[0m[0mcopy[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/ccdproc/core.py[0m in [0;36msubtract_bias[0;34m(ccd, master)[0m
[1;32m    579[0m [0;34m[0m[0m
[1;32m    580[0m     [0;32mtry[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 581[0;31m         [0mresult[0m [0;34m=[0m [0mccd[0m[0;34m.[0m[0msubtract[0m[0;34m([0m[0mmaster[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    582[0m     [0;32mexcept[0m [0mValueError[0m [0;32mas[0m [0me[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    583[0m         [0;32mif[0m [0;34m'operand units'[0m [0;32min[0m [0mstr[0m[0;34m([0m[0me[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/astropy/nddata/ccddata.py[0m in [0;36minner[0;34m(self, operand, operand2, **kwargs)[0m
[1;32m     52[0m             [0;32mglobal[0m [0m_config_ccd_requires_unit[0m[0;34m[0m[0;34m[0m[0m
[1;32m     53[0m             [0m_config_ccd_requires_unit[0m [0;34m=[0m [0;32mFalse[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 54[0;31m             result = self._prepare_then_do_arithmetic(op, operand,
[0m[1;32m     55[0m                                                       operand2, **kwargs)
[1;32m     56[0m             [0;31m# Wrap it again as CCDData so it checks the final unit.[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/astropy/nddata/mixins/ndarithmetic.py[0m in [0;36m_prepare_then_do_arithmetic[0;34m(self_or_cls, operation, operand, operand2, **kwargs)[0m
[1;32m    612[0m [0;34m[0m[0m
[1;32m    613[0m         [0;31m# Now call the _arithmetics method to do the arithmetics.[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 614[0;31m         [0mresult[0m[0;34m,[0m [0minit_kwds[0m [0;34m=[0m [0moperand[0m[0;34m.[0m[0m_arithmetic[0m[0;34m([0m[0moperation[0m[0;34m,[0m [0moperand2[0m[0;34m,[0m [0;34m**[0m[0mkwargs[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    615[0m [0;34m[0m[0m
[1;32m    616[0m         [0;31m# Return a new class based on the result[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/astropy/nddata/mixins/ndarithmetic.py[0m in [0;36m_arithmetic[0;34m(self, operation, operand, propagate_uncertainties, handle_mask, handle_meta, uncertainty_correlation, compare_wcs, **kwds)[0m
[1;32m    244[0m         [0;31m# Then calculate the resulting data (which can but not needs to be a[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[1;32m    245[0m         [0;31m# quantity)[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 246[0;31m         [0mresult[0m [0;34m=[0m [0mself[0m[0;34m.[0m[0m_arithmetic_data[0m[0;34m([0m[0moperation[0m[0;34m,[0m [0moperand[0m[0;34m,[0m [0;34m**[0m[0mkwds2[0m[0;34m[[0m[0;34m'data'[0m[0;34m][0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    247[0m [0;34m[0m[0m
[1;32m    248[0m         [0;31m# Determine the other properties[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/astropy/nddata/mixins/ndarithmetic.py[0m in [0;36m_arithmetic_data[0;34m(self, operation, operand, **kwds)[0m
[1;32m    319[0m                                operand.data * dimensionless_unscaled)
[1;32m    320[0m         [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 321[0;31m             result = operation(self.data * self.unit,
[0m[1;32m    322[0m                                operand.data * operand.unit)
[1;32m    323[0m [0;34m[0m[0m

[0;32m~/Documentos/Docencia/TEA/teaenv/lib64/python3.8/site-packages/astropy/units/quantity.py[0m in [0;36m__array_ufunc__[0;34m(self, function, method, *inputs, **kwargs)[0m
[1;32m    475[0m [0;34m[0m[0m
[1;32m    476[0m         [0;31m# Call our superclass's __array_ufunc__[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 477[0;31m         [0mresult[0m [0;34m=[0m [0msuper[0m[0;34m([0m[0;34m)[0m[0;34m.[0m[0m__array_ufunc__[0m[0;34m([0m[0mfunction[0m[0;34m,[0m [0mmethod[0m[0;34m,[0m [0;34m*[0m[0marrays[0m[0;34m,[0m [0;34m**[0m[0mkwargs[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    478[0m         [0;31m# If unit is None, a plain array is expected (e.g., comparisons), which[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[1;32m    479[0m         [0;31m# means we're done.[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m

[0;31mValueError[0m: operands could not be broadcast together with shapes (760,740) (2012,2040) 
ValueError: operands could not be broadcast together with shapes (760,740) (2012,2040) 

